packages/salesforcedx-sobjects/test/integration/sobjectDescribeTest.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  CommandOutput,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import { expect } from 'chai';
+import * as jsforce from 'jsforce';
+import { SObjectDescribe } from '../../src/describe/sObjectDescribe';
+
+describe('Fetch sObjects', function() {
+  // tslint:disable-next-line:no-invalid-this
+  this.timeout(10000);
+  let username: string;
+  const sobjectdescribe = new SObjectDescribe();
+  const scratchDefFilePath =
```

Comment:
Per our conversation offline, take a look at using `path.join` and also `__dirname`.